stdparams.go

```diff
@@ -0,0 +1,38 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+type paramsKey struct{}
```

Comment:
Why allocate extra memory for an int when this is not needed? See https://play.golang.org/p/Nl61l_h_8-.